registry.json

```diff
@@ -1727,5 +1727,20 @@
       "telegram":"manizzler",
       "operatorName":"manizzler",
       "github":"manizzle"
+   },
+   {
+      "ticker":"ADATA",
+      "poolId":"eb6e1949ef2e1f2cdb9b041d2a4b86d4d837798ab3ddfe53cc4cafe4",
+      "social":{
+         "facebook":"",
+         "twitter":"https://twitter.com/adapooldata",
+         "website":"https://www.pooldata.live",
+         "reddit":"",
+         "instagram":"",
+         "youtube":""
+      },
+      "telegram":"pooldata",
```

Comment:
telegram field can't be a user group.  Please provide a username as SPA voting is done using Telegram.